apps/condo/migrations/20221018154020-0177_alter_multipayment_user.js

```diff
@@ -0,0 +1,30 @@
+// auto generated by kmigrator
+// KMIGRATOR:0177_alter_multipayment_user:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMC40IG9uIDIwMjItMTAtMTggMTA6NDAKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKaW1wb3J0IGRqYW5nby5kYi5tb2RlbHMuZGVsZXRpb24KCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAxNzZfYWx0ZXJfdGlja2V0c291cmNlX3R5cGUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWx0ZXJGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nbXVsdGlwYXltZW50JywKICAgICAgICAgICAgbmFtZT0ndXNlcicsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5Gb3JlaWduS2V5KGJsYW5rPVRydWUsIGRiX2NvbHVtbj0ndXNlcicsIG51bGw9VHJ1ZSwgb25fZGVsZXRlPWRqYW5nby5kYi5tb2RlbHMuZGVsZXRpb24uUFJPVEVDVCwgcmVsYXRlZF9uYW1lPScrJywgdG89J19kamFuZ29fc2NoZW1hLnVzZXInKSwKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Alter field user on multipayment
+--
+SET CONSTRAINTS "MultiPayment_user_2d581ca2_fk_User_id" IMMEDIATE; ALTER TABLE "MultiPayment" DROP CONSTRAINT "MultiPayment_user_2d581ca2_fk_User_id";
+ALTER TABLE "MultiPayment" ALTER COLUMN "user" DROP NOT NULL;
+ALTER TABLE "MultiPayment" ADD CONSTRAINT "MultiPayment_user_2d581ca2_fk_User_id" FOREIGN KEY ("user") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+COMMIT;
```

Comment:
Multi payment is a big table, altering column might take a long time. It's best to disable timeouts for migrations -- please check previous migrations